drivers/serial/Kconfig.stm32

```
@@ -39,8 +39,8 @@ if UART_STM32U5_ERRATA_DMAT_AFFECTED
 
 choice UART_STM32U5_ERRATA_DMAT
 	prompt "Workaround for DMAT errata on selected devices"
-	default UART_STM32U5_ERRATA_DMAT_LOWPOWER if PM
-	default UART_STM32U5_ERRATA_DMAT_NOCLEAR if !PM
+	default UART_STM32U5_ERRATA_DMAT_NOCLEAR if !PM && !POWEROFF
```

`!POWEROFF` shouldn't be needed because the choice symbol already depends on `!POWEROFF`.
```suggestion
	default UART_STM32U5_ERRATA_DMAT_NOCLEAR if !PM
```

---

My mistake. You're right. `if !PM` is enough since the choice will fallback to the default value below when `CONFIG_POWEROFF` is enable making `CONFIG_UART_STM32U5_ERRATA_DMAT_NOCLEAR` not visible.

---

done